docs/docs/add-a-manifest-file.md

```diff
@@ -41,6 +41,9 @@ npm install --save gatsby-plugin-manifest
         // see https://developers.google.com/web/fundamentals/web-app-manifest/#display
         display: "standalone",
         icon: "src/images/icon.png", // This path is relative to the root of the site.
+        crossOrigin: `use-credentials`, // Anything else will be treated as `anonymous`.
```

Comment:
```suggestion
        crossOrigin: `use-credentials`
```

Reply:
maybe to the same thing as display: "standalone":

```
// see https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes
crossOrigin: `use-credentials`,
```